@xen-orchestra/web-core/lib/locales/en.json

```
@@ -77,7 +77,9 @@
   "confirm-cancel": "Are you sure you want to cancel?",
   "confirm-delete": "You're about to delete {0}",
   "connected": "Connected",
-  "connections": "Connections",
+  "connecting": "Connecting",
+  "connection": "Connections",
```

```suggestion
  "connections": "Connections",
```
Remember to update all usages in the components